drafts/docker-open-liberty-raspberry-pi.adoc

```diff
@@ -1,60 +1,73 @@
+This means you can’t take our Open Liberty Docker image from Docker Hub, which is designed for an x86 architecture, and run it on the Pi without any configuration to the Docker images. We need to edit the Dockerfiles that define how to build the x86 Liberty Docker image and modify them to build an image for the ARM Raspberry Pi.
 
-Edit the `FROM` target in the Dockerfile to base the Open Liberty image on an link:https://hub.docker.com/r/arm32v7/adoptopenjdk/[official OpenJDK] image that is compatible with ARM.
+=== Open Liberty images
+To start, take a copy of the kernel Dockerfile for the link:https://github.com/OpenLiberty/ci.docker/tree/master/official/latest/kernel/java12/openj9[x86 Liberty Docker image]. This can be used as the basis for custom built images that contain only the features required for a specific application. 
+Edit the `FROM` target in the Dockerfile to base the Open Liberty image on an link:https://hub.docker.com/r/arm32v7/adoptopenjdk/[OpenJDK] image that is compatible with ARM. For example: 
 
 [source]
-FROM arm32v7/adoptopenjdk:11-jre-hotspot
+----
+FROM arm32v7/adoptopenjdk:latest
 ...
 CMD ["/opt/ol/wlp/bin/server", "run", "defaultServer"]
+----
 
-Build this image so it is stored locally. You will now be able to use this image as a base for your applications. Instructions to build a Docker image that has your application and configuration pre-loaded can be found link:https://github.com/OpenLiberty/ci.docker#building-an-application-image[here]. 
+Download the other folders and files in that directory to build this image so it is stored locally. You will now be able to use this image as a base for your applications. Instructions to build a Docker image that has your application and configuration pre-loaded can be found link:https://github.com/OpenLiberty/ci.docker#building-an-application-image[here]. 
 
-=== Liberty Bikes 
-Liberty bikes is an application consisting of four microservices and each image for the services use the link:https://github.com/OpenLiberty/ci.docker/blob/master/official/19.0.0.9/microProfile2/java11/openj9/Dockerfile[open-liberty:microProfile2-java11] as a base. Edit the `FROM` target in this Dockerfile so that it uses the image that you just built. 
+[source]
+docker build -t OpenLibertyRaspberryPi .
+
+=== Launch a sample application
 
+Try out your image with a sample application. To begin:
 
 [source]
-FROM <YOUR IMAGE NAME>
-RUN cp /opt/ol/wlp/templates/servers/microProfile2/server.xml /config/server.xml
+git clone https://github.com/OpenLiberty/sample-getting-started
+cd sample-getting-started 
 
-Build the microProfile2-java11 image so it is also stored locally and can be used for the application. For example,   
+Replace the contents of the `pom.xml` file in your working directory with the contents of link:https://github.com/OpenLiberty/guide-getting-started/blob/master/finish/pom.xml[this file] so it is configured to include the `liberty-maven-plugin`.
```

Comment:
Instead of "this file", just use the name of the file as the link name; eg `[the pom.xml file]`, or something like that.